doc/renaming_table.xml

```diff
@@ -0,0 +1,30 @@
+<Table Align="|l|l|">
```

Comment:
Do you intend to include `doc/renaming_table/xml` in the manual somewhere?

Reply:
Yepp. I haven't looked into the renaming code for ages and I forgot to add a few changes. :grin: